samples/react-native/android/app/src/main/java/io/sentry/reactnative/sample/MainApplication.kt

```diff
@@ -51,9 +51,8 @@ class MainApplication :
     }
 
     private fun initializeSentry() {
-        SentryAndroid.init(this) { options ->
-            // Only options set here will apply to the Android SDK
-            // Options from JS are not passed to the Android SDK when initialized manually
+        RNSentrySDK.init(this) { options ->
+            // Options set here will apply to the Android SDK overriding the ones from `sentry.options.json`
             options.dsn = "https://1df17bd4e543fdb31351dee1768bb679@o447951.ingest.sentry.io/5428561"
             options.isDebug = true
 
```

Comment:
The `beforeSend` callback should not be needed anymore, the `RNSentrySDK.init` should handle the RN defaults internally.

Reply:
Removed with 7522fcebe114a06f11a9421a96539904e4edda2b